### PR TITLE
Add prometheus push gateway

### PR DESCRIPTION
**Pull request by rohit-dimagi**

### CODE REVIEW

File: dependencies.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@
   version: 0.16.2
 - src: cloudalchemy.node-exporter
   version: 0.19.0
+- src: cloudalchemy.pushgateway
```
Comment (sravfeyn) [inline dependencies.yml:10]: We will also need to run/stop/start the service too, yes? Does it need supervisor?

**Comment by sravfeyn** [comment]
Can you also add a locasetting for `PUSHGATEWAY_HOST`, I guess we will install it on every machine and set it to `localhost:port`

File: dependencies.yml (hunk 1 above)
Comment (rohit-dimagi) [reply dependencies.yml:10]: It will work with systemd.

**Comment by rohit-dimagi** [comment]
> Can you also add a locasetting for `PUSHGATEWAY_HOST`, I guess we will install it on every machine and set it to `localhost:port`

I was thinking one gateway for celery and one for pillow, Also localsetting will be added in commcare-cloud repo

File: roles/prometheus/templates/targets/pushgateway_exporter_target.json.j2 [hunk 2]
```diff
@@ -0,0 +1,27 @@
+[
```
Comment (sravfeyn) [inline roles/prometheus/templates/targets/pushgateway_exporter_target.json.j2:1]: I think we should have `honour_labels: true` [per this](https://github.com/prometheus/pushgateway#about-the-job-and-instance-labels)?

**Comment by rohit-dimagi** [comment]
closing this.

**Pull request closed without merging**